README.md

```diff
@@ -119,6 +119,13 @@ pass the `--post-result` flag:
 $ nixpkgs-review pr --post-result 37242
 ```
 
+If you'd like to include log snippets for failed builds, add the `--with-logs`
+flag:
+
+```console
+$ nixpkgs-review pr --post-result --with-logs 37242
```

Comment:
What do you think about making this the default and we have instead an option that exclude logs or no option at all?

Reply:
I am wondering if this won't affect [r-ryantm](https://github.com/NixOS/nixpkgs/issues?q=is%3Apr+is%3Aopen+author%3Ar-ryantm) updates that seem to include the output of the tool. Since in their case they add more snippets to the final comment, they may overflow the max size of the comment. So having a way to disable seems needed. Whether it's ok to ask users to update their invocations - I don't know, what do you think?

Reply:
Disabling seems to fine. We can also do the same than in r-ryantm, cc @zowoq

Reply:
One way for them to deal with it could be - posting the starting PR description without nixpkgs-review output; then dump report in the second comment. Then they won't risk an overflow.